test/src/responder/gasQueue.test.ts

```diff
@@ -445,7 +446,7 @@ describe("GasQueue", () => {
         expect(uQ.difference(q)).to.deep.equal([]);
     })
 
-    it("unlock does throws error for missing nonces", () => {
+    it("unlock does throw error for missing nonces", () => {
```

Comment:
This should be `prepend` - the function used to be called `unlock` but refactoring failed here.